app/src/main/res/values/strings.xml

```diff
@@ -2,5 +2,14 @@
     <string name="app_name">Teckers App</string>
     <string name="text">Text</string>
     <string name="logOut">Log Out</string>
+    <string name="title_messages">Messages</string>
```

Comment:
I don't know if the final user will use the app in English. If not, please add a `strings.xml` file for Spanish.

Reply:
I will change them into a spanish file for the strings. I will call it spanishString.xml

Reply:
That is not the correct way to do it, check this out:
https://developer.android.com/guide/topics/resources/localization